tests_live/test_structure.py

```diff
@@ -1,4 +0,0 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
-def test_explosions():
-    pass
```

Comment:
I hope you found the existence of this file as funny as I did

Reply:
😆